pkg/cli/list.go

```diff
@@ -133,6 +138,8 @@ defined`,
 	// Use PersistentFlags() for flags inherited by subcommands or Flags() for local flags.
 	c.Flags().StringVarP(&focusWorkload, "focusworkload", "", "",
 		"Focus connections of specified workload in the output (<workload-name> or <workload-namespace/workload-name>)")
+	c.Flags().StringVarP(&focusDirection, "focus-direction", "", "",
+		"Filter connlist of specified workload for input direction {ingress, egress, both (default both)}")
```

Comment:
```suggestion
		"Focus connections of specified workload on one direction, applies only when focusworkload is used" {ingress/egress}
```

Reply:
done + change this flag so it has only [ingress /egress] values in code